packages/js-drive/lib/abci/handlers/initChainHandlerFactory.js

```diff
@@ -64,17 +67,17 @@ function initChainHandlerFactory(
 
     await rsAbci.initChain({ }, true);
 
-    // Create misc tree
-    await groveDBStore.createTree(
-      [],
-      Buffer.from([5]),
-      { useTransaction: true },
+    const blockInfo = new BlockInfo(
+      0,
+      0,
+      protoTimestampToMillis(time),
     );
```

Comment:
We don't need to create tree here anymore?

Reply:
Yes, now it's RS Drive's responsibility.